src/TextInput/PasswordInput.svelte

```diff
@@ -88,6 +88,7 @@
 
   const ctx = getContext("Form");
 
+  // svelte-ignore reactive_declaration_non_reactive_property
   $: isFluid = !!ctx && ctx.isFluid;
```

Comment:
The warning here looks valid.
Contexts are fixed on component creation and properties themselves do not trigger reactivity, so this statement will only run once => could just be a regular variable.
```suggestion
  const isFluid = !!ctx && ctx.isFluid;
```